src/modules/scraper/adapter/messaging/FillEventsConsumer.ts

```diff
@@ -33,6 +33,10 @@ export class FillEventsConsumer {
     }
 
     await this.processFillEventQueueMessage(deposit, job.data);
+
+    this.scraperQueuesService.publishMessage<DepositFilledDateQueueMessage>(ScraperQueue.DepositFilledDate, {
+      depositId: deposit.id,
+    });
```

Comment:
after the Fill event is processed by this consumer, send another message to `DepositFilledDate` queue for computing the filled date